syntax = "proto3";

package StreamServer;


message TestMessage{
  string message = 1;
}
message TestRequest{}


message TestStreamMessage{
  string message = 1;
}
message TestStreamRequest{}


service StreamService {
  rpc testFunc(TestRequest) returns (TestMessage){}
  rpc testStreamFunc(TestStreamRequest) returns (stream TestStreamMessage){}
}
